src/Components/BottomLayout/BottomLayout.js

```diff
@@ -1,67 +1,18 @@
-import { ReactComponent as Blue } from '../../Assets/blueWave.svg';
-import { ReactComponent as Lightgreen } from '../../Assets/greenFaded.svg';
-import { ReactComponent as Green } from '../../Assets/greenWave.svg';
-import { ReactComponent as Lightpurple } from '../../Assets/purpleFaded.svg';
-import { ReactComponent as Purple } from '../../Assets/purpleStrong.svg';
 import styled from 'styled-components/macro';
+import { ReactComponent as BottomDesign } from '../../Assets/bottomDesign.svg';
```

Comment:
I can't think of a better name but I have some issues with the name BottomDesign :D It is not a big issue but maybe you find a better name at one point. I'll also think about it.